### ISSUE TITLE
Issue using clipped on Windows 10 box

### ISSUE THREAD

**Issue by devtech8** [clippedjs/clipped#118]
### Description

Installs, but when tying t use `create` or `init`, it errors.

```
λ clipped create
module.js:550
    throw err;
    ^

Error: Cannot find module '../dist/src'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\Mark\AppData\Roaming\npm\node_modules\clipped\bin\clipped:3:1)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

Using node 8.12 and npm 6.4.1

### Suggestions

-

**Comment by IniZio** [comment]
Thanks for your contrib :)

Resolved in v2.1.4, will close this when confirmed resolved.

**Comment by IniZio** [comment]
I will close this issue and open another issue for `create` command error, thanks anyways for the report :)
